source/security/openid-external-identity-management/external-authentication-with-openid-identity-provider.rst

```diff
@@ -126,10 +126,11 @@ provider for instructions on configuring user claims.
 
 MinIO provides :ref:`built-in policies <minio-policy-built-in>` for basic access
 control. You can create new policies using the :mc:`mc admin policy` command, or
-by using the MinIO Console. MinIO does not support assigning :ref:`groups
-<minio-groups>` to an :abbr:`OIDC (OpenID Connect)` managed identity. Specify
-any and all policies to attach to the user as part of its :abbr:`JWT (JWT)`
-policy claim.
+by using the MinIO Console. 
+
+MinIO does not support assigning an identity managed by :abbr:`OIDC (OpenID Connect)` to a MinIO :ref:`group <minio-groups>`. 
+Instead, an :abbr:`OIDC (OpenID Connect)` administrator can use the user-specified claim to list multiple MinIO :ref:`policies <minio-policy>` to assign to the user.
+In this way, the OIDC administrator can create a type of "group" assignment managed entirely within :abbr:`OIDC (OpenID Connect)`.
```

Comment:
Style point: "In this way" doesn't add anything. We can cut to the point and just say:

```suggestion
The OIDC administrator can use this method to mimic "group" assignment managed entirely within :abbr:`OIDC (OpenID Connect)`.
```

Reply:
👍